demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -40,7 +40,7 @@ class ContentCreator:
 
     def __init__(self, artifacts_path: str, content_version='', content_bundle_path='',
                  test_bundle_path='', packs_bundle_path='', preserve_bundles=False, packs=False,
-                 no_update_commonserver=False):
+                 no_update_commonserver=False, packs_to_include=[]):
```

Comment:
```suggestion
                 no_update_commonserver=False, packs_to_include=''):
```
default list might cause problems